### DISCUSSION TITLE
is it possible to install this as an openfx plugin for davinci resolve?

### DISCUSSION THREAD

**Discussion by doomgitdoom3** [ntsc-rs/ntsc-rs#127] [General]
The title. 

Ive been trying to figure out how to install this into davinci resolve. How do i do this?

**Comment by valadaptive** [comment]
Copy `NtscRs.ofx.bundle` to `C:\Program Files\Common Files\OFX\Plugins`

**Reply by ryanskim04** [reply]
I don't see a OFX folder in my common files

**Reply by valadaptive** [reply]
Create one, then
